sdk/python/kfp/components/executor_test.py

```diff
@@ -219,13 +356,13 @@ def test_function_string_output(self):
       "inputs": {
         "parameterValues": {
           "first_message": "Hello",
-          "second_message": "",
+          "second_message": ", ",
           "third_message": "World"
         }
       },
       "outputs": {
         "parameters": {
-          "output": {
+          "Output": {
```

Comment:
Curious why do we have this diff? I guess executor input json is case-insensitive, you made the change so that it's more "correct" as KFP use `Output` as single return output name?

Reply:
That's exactly it: when a component of this type is executed this is what the executor input looks like.